### PR TITLE
Bump money gem dependency from 6.13 to 6.16

### PR DESCRIPTION
**Pull request by alistairholt**

### CODE REVIEW

**Comment by alistairholt** [comment]
Can someone with write access review this? The tests pass locally.

**Comment by bkubic** [comment]
Bumping this - we're blocked on implementing this gem until the dependency is resolved. Any chance this could get merged soon?

**Review by semmons99 — APPROVED** [review]

**Pull request merged**
